File: sdk/csharp/libraries/microsoft.bot.builder.solutions/Dialogs/RouterDialogTurnResult.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

namespace Microsoft.Bot.Builder.Solutions.Dialogs
{
    /// <summary>
    /// Define router dialog turn result.
    /// </summary>
    public class RouterDialogTurnResult
    {
        public RouterDialogTurnResult(RouterDialogTurnStatus status)
        {
            this.Status = status;
        }

        public RouterDialogTurnStatus Status { get; set; }
    }
}

File: sdk/typescript/libraries/botbuilder-solutions/src/dialogs/routerDialogTurnResult.ts
/**
 * Copyright(c) Microsoft Corporation.All rights reserved.
 * Licensed under the MIT License.
 */

import { RouterDialogTurnStatus } from './routerDialogTurnStatus';

/**
 * Define router dialog turn result.
 */
export class RouterDialogTurnResult {

    public status: RouterDialogTurnStatus;

    public constructor (status: RouterDialogTurnStatus) {
        this.status = status;
    }
}
